app/routes/admin/sales/index.js

```diff
@@ -1,28 +1,16 @@
 import Route from '@ember/routing/route';
+
 export default Route.extend({
   titleToken() {
     return this.get('l10n').t('Overview');
   },
 
-  model() {
-    return [{
-      events           : 'Sample Event',
-      eventDate        : new Date(),
-      completedTickets : 1,
-      completedSales   : 2.00,
-      placedTickets    : 1,
-      placedSales      : 4.40,
-      pendingTickets   : 1,
-      pendingSales     : 5.00
-    }, {
-      events           : 'Sample Event',
-      eventDate        : new Date(),
-      completedTickets : 3,
-      completedSales   : 23.30,
-      placedTickets    : 4,
-      placedSales      : 32.00,
-      pendingTickets   : 4,
-      pendingSales     : 32.00
-    }];
+  async model() {
+    let filterOptions = [];
+
+    return this.store.query('admin-sales-by-event', {
+      filter       : filterOptions,
+      'page[size]' : 10
```

Comment:
Don't limit the page size to 10, rather for now define it as 0 to fetch all the records as admin sales table does not has pagination yet. If you like, you can work on converting it to an ember model table after this PR is merged.